package.json

```diff
@@ -15,54 +19,56 @@
 	],
 	"dependencies": {
 		"classnames": "^2.3.1",
-		"prop-types": "^15.7.2",
-		"prosemirror-commands": "^1.1.10",
-		"prosemirror-dropcursor": "^1.3.5",
-		"prosemirror-gapcursor": "1.0.2",
-		"prosemirror-history": "^1.1.3",
-		"prosemirror-inputrules": "^1.1.3",
-		"prosemirror-keymap": "^1.1.4",
-		"prosemirror-model": "^1.14.2",
-		"prosemirror-schema-basic": "^1.1.2",
-		"prosemirror-schema-list": "^1.1.4",
-		"prosemirror-state": "^1.3.4",
+		"clean-webpack-plugin": "^4.0.0",
+		"core-js": "^3.23.3",
+		"css-minimizer-webpack-plugin": "^4.0.0",
+		"mini-css-extract-plugin": "^2.6.1",
+		"postcss": "^8.4.14",
+		"prop-types": "^15.8.1",
+		"prosemirror-commands": "^1.3.0",
+		"prosemirror-dropcursor": "^1.5.0",
+		"prosemirror-gapcursor": "^1.3.1",
+		"prosemirror-history": "^1.3.0",
+		"prosemirror-inputrules": "^1.2.0",
+		"prosemirror-keymap": "^1.2.0",
+		"prosemirror-model": "^1.18.1",
+		"prosemirror-schema-list": "^1.2.0",
+		"prosemirror-state": "^1.4.1",
 		"prosemirror-tables": "^0.9.5",
-		"prosemirror-transform": "^1.3.2",
+		"prosemirror-transform": "^1.6.0",
 		"prosemirror-utils": "^0.9.6",
-		"prosemirror-view": "^1.18.8",
+		"prosemirror-view": "^1.26.2",
 		"react": "^16.14.0",
 		"react-dom": "^16.14.0",
-		"react-intl": "^5.20.4"
+		"react-intl": "^6.0.4",
+		"terser-webpack-plugin": "^5.3.3",
+		"webpack-config-dump-plugin": "^3.0.1"
```

Comment:
Packages that are only used for the build process should stay in `devDependencies`.